src/doplarr/backends/radarr.clj

```diff
@@ -19,16 +19,20 @@
     (let [quality-profiles (a/<! (impl/quality-profiles))
           rootfolders (a/<! (impl/rootfolders))
           {:keys [radarr/quality-profile]} @state/config
-          default-profile-id (utils/id-from-name quality-profiles quality-profile)]
+          default-profile-id (utils/id-from-name quality-profiles quality-profile)
+          default-root-folder (utils/id-from-name rootfolders rootfolders)]
```

Comment:
You need to grab the rootfolder selection out of `@state/config` because `(utils/id-from-name rootfolders rootfolders)` won't do anything without a rootfolder to choose from. It'd be like

```clj
(let [quality-profiles (a/<! (impl/quality-profiles))
          rootfolders (a/<! (impl/rootfolders))
          {:keys [radarr/quality-profile radarr/rootfolder]} @state/config
          default-profile-id (utils/id-from-name quality-profiles quality-profile)
          default-root-folder (utils/id-from-name rootfolders rootfolder)]
...)
```